app/views/relatives/_show.html.slim

```diff
@@ -7,5 +7,5 @@ tr
           dt= rel.full_name
           - if rel.full_name.present?
             dd
-              = l(rel.date_of_birth) if rel.date_of_birth
+              = rel.date_of_birth.year if rel.date_of_birth
```

Comment:
`rel.date_of_birth.try(:year)` and as you are working in this file, can you fix `rel` -> `relative`